inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -178,12 +179,23 @@ ie::CNNNetwork FrontEnd::convertNetwork(ie::CNNNetwork& network) {
         ngraph::opset5::Convolution::type_info,
         ngraph::opset5::GroupConvolution::type_info
     });
-
+    const std::vector<std::pair<ngraph::element::Type, ngraph::element::Type>> convert_precision_list {
+        {ngraph::element::i64, ngraph::element::i32},
+        {ngraph::element::u64, ngraph::element::i32},
+        {ngraph::element::u32, ngraph::element::i32},
+        {ngraph::element::u16, ngraph::element::i32},
+        {ngraph::element::boolean, ngraph::element::i32},
+    };
+    for (const auto &precision : convert_precision_list) {
+        manager.register_pass<ngraph::pass::ConvertPrecision>(precision.first, precision.second, myriad_type_to_fuse);
```

Comment:
Using ConvertPrecision you need to remember that in cases when operation doesn't support some particular input/output type we use `TypeRelax` wrapper that may affect transformations that are based on some maps like this `map<NodeTypeInfo, ...>` (if you use MathcerPass or FunctionPass with dynamic_casts everything will be ok).